management/client.go

```diff
@@ -303,6 +449,59 @@ type SAML2ClientAddonLogout struct {
 	SLOEnabled *bool `json:"slo_enabled,omitempty"`
 }
 
+// SAPAPIClientAddon defines the `sap` settings for a client.
+type SAPAPIClientAddon struct {
+	ClientID             *string `json:"clientid,omitempty"`
+	UsernameAttribute    *string `json:"usernameAttribute,omitempty"`
+	TokenEndpointURL     *string `json:"tokenEndpointUrl,omitempty"`
+	Scope                *string `json:"scope,omitempty"`
+	ServicePassword      *string `json:"servicePassword,omitempty"`
+	NameIdentifierFormat *string `json:"nameIdentifierFormat,omitempty"`
+}
+
+// SentryClientAddon defines the `sentry` settings for a client.
+type SentryClientAddon struct {
+	OrgSlug *string `json:"org_slug,omitempty"`
+	BaseURL *string `json:"base_url,omitempty"`
+}
+
+// SharepointClientAddon defines the `sharepoint` settings for a client.
+type SharepointClientAddon struct {
+	URL         *string   `json:"url,omitempty"`
+	ExternalURL *[]string `json:"external_url,omitempty"`
+}
+
+// SlackClientAddon defines the `slack` settings for a client.
+type SlackClientAddon struct {
+	Team *string `json:"team,omitempty"`
+}
+
+// SpringCMClientAddon defines the `springcm` settings for a client.
+type SpringCMClientAddon struct {
+	Acsurl *string `json:"acsurl,omitempty"`
```

Comment:
Should this property be `AcsURL` instead?

Reply:
Or `ACSURL`?

Reply:
Fixed to be `ACSURL`